test-app/tests/helpers/index.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from 'ember-qunit';
 
 export interface CustomTestContext extends RenderingTestContext {
```

Comment:
Shouldn't this be only added if the `setupStylesheetSupport` function is called? But I think we can address this independently of this PR as it's not an issue introduced by it but existing before.